scripts/update_version.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
```

Comment:
A small thing, but wouldn't calling this file `bump_version.sh` instead of `update_version.sh` be more conventional?